pep-0011.txt

```diff
@@ -14,109 +14,116 @@ Post-History: 18-Aug-2007,
-  removed
+Platform support is broken down into *tiers*. Each tier comes with
+different requirements which lead to different promises being made
+about support.
 
-In some cases, it is not possible to identify the specific list of
-systems for which some code is used (e.g. when autoconf tests for
-absence of some feature which is considered present on all
-supported systems).  In this case, the name will give the precise
-condition (usually a preprocessor symbol) that will become
-unsupported.
+Tier 1
+------
 
-At the same time, the CPython source code must be changed to
-produce a build-time error if somebody tries to install Python on
-this platform.  On platforms using autoconf, configure must fail.
-This gives potential users of the platform a chance to step
-forward and offer maintenance.
+- `CI failures <https://github.com/python/cpython/actions/workflows/build.yml?query=branch%3Amain+is%3Acompleted>`__ block releases.
+- Changes which would break the ``main`` branch are not allowed to be merged;
+  any breakage may be reverted immediately.
+- All core developers are responsible to keep these platforms,
+  and thus ``main``, working.
+- Promotion to this tier requires team consensus/SC approval.
+
+====================== =====
+Target Triple          Notes
+====================== =====
+i686-pc-windows-msvc
+x86_64-pc-windows-msvc
+x86_64-apple-darwin    BSD libc, clang
+x86_64-\*-linux-gnu    glibc, gcc
+====================== =====
+
+
+Tier 2
+------
+
+- Must have a reliable buildbot.
+- At least **two** core developers are signed up to support the platform.
+- Changes which break any of these platforms are to be fixed or
+  reverted within 24 hours.
+- Failures on these platforms block a release.
+- Promotion to this tier requires consensus/SC approval.
+
+====================== ========================== ============================================== ========
+Target Triple          Notes                      Buildbot                                       Contacts
+====================== ========================== ============================================== ========
+aarch64-apple-darwin   clang                      https://buildbot.python.org/all/#/builders/725 XXX
+aarch64-linux-gnu      glibc, gcc                 https://buildbot.python.org/all/#/builders/125 XXX
+
+                       glibc, clang               https://buildbot.python.org/all/#/builders/234 XXX
+aarch64-windows-msvc                              https://buildbot.python.org/all/#/builders/729 XXX
+powerpcle-linux-gnu    glibc, gcc                 https://buildbot.python.org/all/#/builders/90  XXX
+
+                       glibc, clang               https://buildbot.python.org/all/#/builders/435 XXX
+s390x-linux-gnu        glibc, gcc                 https://buildbot.python.org/all/#/builders/223 XXX
+
+                       glibc, clang               https://buildbot.python.org/all/#/builders/3   XXX
```

Comment:
Sadly, I don't have access to s390x buildbot workers, so it's harder for me to debug such issue :-( Maybe move it to Tier 3.